frontend/src/pages/org/settings/components/crawl-workflows.ts

```diff
@@ -0,0 +1,336 @@
+          value=${ifDefined(orgDefaults.behaviorTimeout)}
+          placeholder=${defaultLabel(this.defaults.behaviorTimeoutSeconds)}
+          min="0"
+        >
+          <span slot="suffix">${msg("seconds")}</span>
+        </sl-input>
+      `,
+      pageExtraDelaySeconds: html`
+        <sl-input
+          size="small"
+          name="pageExtraDelaySeconds"
+          type="number"
+          inputmode="numeric"
+          label=${msg("Delay Before Next Page")}
+          value=${ifDefined(orgDefaults.pageExtraDelay)}
+          placeholder=${defaultLabel(0)}
+          min="0"
+        >
+          <span slot="suffix">${msg("seconds")}</span>
+        </sl-input>
+      `,
+    };
+    const browserSettings = {
+      browserProfile: html`
+        <btrix-select-browser-profile
+          profileId=${ifDefined(orgDefaults.profileid)}
+          size="small"
+        ></btrix-select-browser-profile>
+      `,
+      crawlerChannel: html`
+        <btrix-select-crawler
+          crawlerChannel=${ifDefined(orgDefaults.crawlerChannel)}
+          size="small"
+        ></btrix-select-crawler>
+      `,
+      blockAds: html`<sl-checkbox
+        size="small"
+        name="blockAds"
+        ?checked=${orgDefaults.blockAds ?? true}
+      >
+        ${msg("Block ads by domain")}
+      </sl-checkbox>`,
+      userAgent: html`
+        <sl-input
+          size="small"
+          name="userAgent"
+          label=${msg("User Agent")}
+          value=${ifDefined(orgDefaults.userAgent)}
+          autocomplete="off"
+          placeholder=${msg("Default: Browser User Agent")}
+        >
+        </sl-input>
+      `,
+      lang: html`
+        <btrix-language-select
+          value=${ifDefined(
+            orgDefaults.lang ? (orgDefaults.lang as LanguageCode) : undefined,
+          )}
+          size="small"
+          @on-change=${(e: CustomEvent<{ value: string | undefined }>) => {
+            console.log(e.detail.value);
+          }}
+        >
+          <span slot="label">${msg("Language")}</span>
+        </btrix-language-select>
+      `,
+    };
+
+    return {
+      scope,
+      perCrawlLimits,
+      perPageLimits,
+      browserSettings,
+    };
+  }
+
+  private renderWorkflowDefaults() {
+    return html`
+      <div class="rounded-lg border">
+        <form @submit=${this.onSubmit}>
+          ${guard([this.defaults, this.org], () =>
+            Object.entries(this.fields).map(([sectionName, fields]) =>
+              section(
+                sectionName as SectionsEnum,
+                Object.entries(fields).map(([fieldName, field]) => [
+                  field,
+                  infoTextStrings[fieldName as FieldName],
+                ]),
+              ),
+            ),
+          )}
+          <footer class="flex justify-end border-t px-4 py-3">
+            <sl-button type="submit" size="small" variant="primary">
+              ${msg("Save Changes")}
+            </sl-button>
+          </footer>
```

Comment:
Hardly a blocker, but it could be nice to also have a "Reset" button here somewhere, perhaps with dialog confirmation before applying. Right now if you want to clear all the values and start over it's a lot of clicks/manual deletion.

Reply:
Would that reset to the previously saved defaults, or remove all defaults altogether? The former can just be accomplished by refreshing the page..
It seems like users might be changing these incrementally, so maybe not too big of a deal either way.

Reply:
I was imagining a function just to clear the form altogether, but it's true that in practice that probably wouldn't be done much so unless we get user requests, just ignore this!